tests/PartnerClientTest.php

```diff
@@ -65,6 +66,29 @@ public function testChangeSeatsHappyPath()
             self::fail('error on ChangeSeats: ' . $e);
         }
 
-        self::assertNotEmpty($resp, 'expected a response');
+        self::assertEmpty($resp, 'expected an empty response');
+    }
+
+    public function testUpdateSSOHappyPath()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $partnerClient = new PartnerClient($environment);
+
+        $req = new UpdateSSORequest();
+        $req->setDomain("google.com");
+        // disable SSO
+        $req->setEnableSso(false);
```

Comment:
probably too late for this, but the wording around this endpoint is a bit weird `setEnableSSO` and `false` doesn't make a lot of sense

Reply:
I think the EnableSSO field itself makes sense as long as users are aware that this is a setter method for a boolean field

Reply:
Oh right, I forgot this was the field 👍 Ok gotcha, that makes sense I just misread that before